edge-apps/countdown-timer/screenly_qc.yml

```diff
@@ -9,38 +9,38 @@ settings:
   countdown_headline:
     type: string
     default_value: New Year Countdown
-    title: countdown_headline
+    title: Countdown Headline Text
     optional: false
     help_text: Countdown header text value.
   enable_analytics:
     type: string
     default_value: 'true'
-    title: enable_analytics
+    title: Enable Analytics
     optional: true
     help_text: Whether to disable Sentry and Google Analytics integrations
     is_global: true
   override_locale:
     type: string
     default_value: en
-    title: override_locale
+    title: Override locale
     optional: true
     help_text: Specify a supported locale value (e.g., en_US).
   override_timezone:
     type: string
     default_value: ''
-    title: override_timezone
+    title: Override Timezone
     optional: true
     help_text: Specify a supported timezone value (e.g., Europe/London).
   tag_manager_id:
     type: string
     default_value: GTM-P98SPZ9Z
-    title: tag_manager_id
+    title: Tag Manager ID
     optional: true
     help_text: Specify a Google Tag Manager ID
     is_global: true
   target_timestamp:
     type: string
     default_value: 2025-01-01T00:00:00
-    title: target_timestamp
+    title: Target Countdown Date & Time (Format YYYY-MM-DDTHH:MM:SS).
```

Comment:
Don't add Format in the title, it's should be in the description

Reply:
Okay. Sure.